book/P3C14_forests.qmd

```diff
@@ -0,0 +1,310 @@
+
+Other rules have also been studied including comparison of residuals  [@Therneau1990], scoring rules [@pkgrfsrc], distance metrics [@Gordon1985], and concordance metrics [@Schmid2016].
+Experiments have shown different splitting rules may perform better or worse depending on the underlying data [@Schmid2016], hence one could even consider treating the splitting rule as a hyper-parameter for tuning.
+However, if there is a clear goal in prediction, then the choice of splitting rule can be informed by the prediction type.
+For example, if the goal is to maximise separation, then a log-rank splitting rule to maximise homogeneity in terminal nodes is a natural starting point.
+Whereas if the goal is to accurately rank observations, then a concordance splitting rule may be optimal.
+
+### Terminal Node Prediction {#sec-surv-ml-models-ranfor-nodes}
+
+As in the regression setting, the usual strategy for predictions is to create a simple estimate based on the training data that lands in the terminal nodes.
+However, as seen throughout this book, the choice of estimator in the survival setting depends on the prediction task of interest, which are now considered in turn.
+First, note that all terminal node predictions can only yield useful results if there are a sufficient number of uncensored observations in each terminal node.
+Hence, a common RSF stopping rule is the minimum number of *uncensored* observations per leaf, i.e., a leaf is not split if that would result in too few uncensored observations in the resulting leaves.
+
+
+#### Probabilistic Predictions {.unnumbered .unlisted}
+
+Starting with the most common survival prediction type, the algorithm requires a simple estimate for the underlying survival distribution in each of the terminal nodes, which can be estimated using the Kaplan-Meier or Nelson-Aalen methods [@Hothorn2004; @Ishwaran2008; @LeBlanc1993; @Segal1988].
+
+<!-- FIXME: REF KM DEFINITION ABOVE -->
+
+By convention, the Nelson-Aalen estimator is used more commonly for RSFs and is thus focused on in this section, but the same logic follows for the Kaplan-Meier estimator.
+
+Once the Nelson-Aalen estimate for the cumulative hazard function is estimated for individual trees, the question is then how how to combine them to create a single random forests prediction.
+As Nelson-Aalen estimates produce discrete hazard functions, the same bagging algorithm can be used relatively simply by averaging the estimated hazard probability at each observed time-point [@Hothorn2004].
+See @fig-surv-ranfor-lung for an example using the `lung` dataset [@pkgsurvival].
+
+Let $b_m \in \{1,...,b_M\}$ be a terminal node in a survival decision tree, $b$, with $b_M \in \PReals$ terminal nodes, then if an observation, $x$, lands in terminal node $b_m$, its predicted cumulative hazard is given by,
```

Comment:
My notation is meant to be saying that M is dependent on b. So tree 'b' has 'b_M' terminal nodes, whereas tree 'c' has 'c_M' terminal nodes. So it should be 1,...,b_M. Let me know if that doesn't make sense... also M \in \mathbb{N} (or N^+ N^* N_1) not \mathbb{N}_0 because can't have 0 terminal node?